src/status_im/extensions/core.cljs

```diff
@@ -691,15 +691,21 @@
     {:errors [{:type type :value value}]}))
 
 (def uri-prefix "https://get.status.im/extension/")
```

Comment:
You can probably have a single regex, but not that it matters that much, more explicit this way